src/common/packager.ts

```diff
@@ -191,28 +195,36 @@ export class Packager {
                     executedStartPackagerCmd = true;
                     return this.monkeyPatchOpnForRNPackager()
                         .then(() => {
-                            let args = ["--port", port.toString()];
+                            let args: any = ["--port", port.toString()];
```

Comment:
`child_process.spawn` is flatted nested arrays, we can get array from `app.json` or `exp.json`